internal/sdk/internal/hooks/credentials_test.go

```diff
@@ -39,7 +39,7 @@ func TestGetCredentialsNoFile(t *testing.T) {
 	os.Setenv("CRIBL_CLIENT_SECRET", "")
 	os.Setenv("CRIBL_ORGANIZATION_ID", "")
 	os.Setenv("CRIBL_WORKSPACE_ID", "")
-	os.Setenv("HOME", "/tmp")
+	os.Setenv("HOME", "/var/tmp")
```

Comment:
most os's won't have an issue with this but you might want to consider `$TMPDIR` or `os.MkdirTemp` to platform-proof the tests a bit but this is just a suggestion.